R/lint.R

```diff
@@ -12,15 +12,27 @@ NULL
 #' Lint a file
 #'
 #' Apply one or more linters to a file and return the lints found.
+#'
 #' @name lint_file
-#' @param filename the given filename to lint.
-#' @param linters a named list of linter functions to apply see \code{\link{linters}}
-#' for a full list of default and available linters.
+#'
+#' @param filename either the filename for a file to lint, or a character
+#' string of inline R code for linting. If a newline character is detected,
```

Comment:
nit: "if" should be "if and only if". I might reword it to something like

> The latter [inline data] applies whenever `filename` has a newline character (`\n`).

Reply:
Fixed